### ISSUE TITLE
Complains about local files

### ISSUE THREAD

**Issue by IanVS** [ryanramage/check-stale-node-modules#2]
npm allows local folders to be added as dependencies (see https://docs.npmjs.com/files/package.json#local-paths).  However, this package will warn that the dependency is not satisfied.  The actual message is:

```
dependency shells does not satisfy file:client/shells
run `npm install`

npm ERR! Darwin 15.2.0
npm ERR! argv "/Users/ianvs/.nvm/versions/node/v5.3.0/bin/node" "/Users/ianvs/.nvm/versions/node/v5.3.0/bin/npm" "run" "clientBuild"
npm ERR! node v5.3.0
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! nutshell@ preclientBuild: `check-stale-node-modules`
npm ERR! Exit status 1
```

My package.json looks like:

```
"dependencies": {
    "shells": "file:client/shells"
  },
```

Any chance you could allow this syntax without throwing an error?

**Comment by ryanramage** [comment]
Interesting, I have never used local-paths, I usually run `npm-link ../client/shells`. Does file:client/shells contain a package.json? 

I think the issue might actually be in the [read-package-tree](https://github.com/npm/read-package-tree) module but for now, I could just ignore file: depends until that is fixed.

**Comment by ryanramage** [comment]
Oh hmm, also it could be that your dependency specification does not contain a version, does this work?

```
"dependencies": {
    "shells": "file:client/shells@1.0.0"
  },
```

where the version you put there is the same as your content/shells package.json

**Comment by IanVS** [comment]
Yes, it has a very minimal package.json.  It's a reusable component library that we are keeping within our git repo, but want to treat it as a separate npm module (for ease of use, so we can `import {Foo, Bar} from 'shells'`).

Unfortunately adding a version seems to have no effect.

**Comment by ryanramage** [comment]
Ok I have published 1.0.3 which should ignore your situation. It was a quick fix, so let me know if that works for you. thx!

**Comment by IanVS** [comment]
That seems to have done it, thanks!
